package.json

```diff
@@ -42,6 +42,7 @@
     "@ng-bootstrap/ng-bootstrap": "^9.1.3",
     "@ngx-translate/core": "^13.0.0",
     "@ngx-translate/http-loader": "^6.0.0",
+    "@tmcw/togeojson": "^4.4.1",
```

Comment:
Couldn't we use the already exisiting OL parsers for gpx and kml?

Reply:
Well, ye it has got to be possible, sure. Thanks. I'll rewrite it .. got this part from Dailis as he had that implemented for whiteboard